tests/equilibrium/test_equilibrium.py

```diff
@@ -600,6 +600,7 @@ def test_circular_eq_netcdf_round_trip(tmp_path, circular_eq):
         ("transp_eq.cdf", "TRANSP"),
         ("transp_eq.geqdsk", "GEQDSK"),
         ("test.geqdsk", "GEQDSK"),
+        ("reverse_ipbt.geqdsk", "GEQDSK"),
```

Comment:
Rather than putting it in the filetype inference test, could this have a new test? I don't think it'd need to be all that complicated, just something like the following:

```python
def test_reverse_ipbt():
    # Check we can handle files with decreasing psi
    eq = read_equilibrium(template_dir / "reverse_ipbt.geqdsk")
    assert(eq.psi(0.0) > eq.psi(1.0))
    assert(eq.I_p == ...) # golden answer, whatever negative value it takes
    assert(eq.f(0.0) == ...) # check a few arrays, make sure they haven't been accidentally reversed
```

Reply:
Added a more rigorous test